1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,128 @@
 // Write your code here
+const photos = [
+  {
+    src: "./assets/alex-bertha-Jyg7xHRmXiU-unsplash.jpg",
+    // src: "./assets/cute-cat-a.png",
+  },
+  {
+    src: "./assets/roman-raizen-xadg_h62oG8-unsplash.jpg",
+  },
+  {
+    src: "./assets/vidar-nordli-mathisen-AvnXCFX25GA-unsplash.jpg",
+  },
+];
+
+
+
+//variables
+const imageNumber = document.querySelector("#image-number");
+const image = document.querySelector("#carousel-img");
+// let photoPosition = 0;
+// photoContent = photos[photoPosition];
+let ids = 0;
+
+let intervalID;
+
+//forward button
+const forwardButton = document.getElementById("forward-btn");
+forwardButton.addEventListener("click", forwardAction);
+
+function forwardAction() {
+  ids++;
+  if (ids>2) {
+    ids=0;
+  }
```

Comment:
Could be more efficient to use a ternary operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator